ublox/src/ubx_packets/packets.rs

```diff
@@ -708,6 +708,240 @@ impl CfgMsgAllPortsBuilder {
     }
 }
 
+/// Navigation Engine Settings
+#[ubx_packet_recv_send]
+#[ubx(
+    class = 0x06,
+    id = 0x24,
+    fixed_payload_len = 36,
+    flags = "default_for_builder"
+)]
+struct CfgNav5 {
+    /// Only the masked parameters will be applied
+    #[ubx(map_type = CfgNav5Params)]
+    mask: u16,
+    #[ubx(map_type = CfgNav5DynModel, may_fail)]
+    dyn_model: u8,
+    #[ubx(map_type = CfgNav5FixMode, may_fail)]
+    fix_mode: u8,
+
+    /// Fixed altitude (mean sea level) for 2D fixmode (m)
+    #[ubx(map_type = f64, scale = 0.01, into = ScaleBack::<f64>(100.).as_i32)]
+    fixed_alt: i32,
+
+    /// Fixed altitude variance for 2D mode (m^2)
+    #[ubx(map_type = f64, scale = 0.0001, into = ScaleBack::<f64>(1e4).as_u32)]
+    fixed_alt_var: u32,
+
+    /// Minimum Elevation for a GNSS satellite to be used in NAV (deg)
+    min_elev_degrees: i8,
+
+    /// Reserved
+    dr_limit: u8,
+
+    /// Position DOP Mask to use
+    #[ubx(map_type = f32, scale = 0.1, into = ScaleBack::<f32>(10.).as_u16)]
+    pdop: u16,
+
+    /// Time DOP Mask to use
+    #[ubx(map_type = f32, scale = 0.1, into = ScaleBack::<f32>(10.).as_u16)]
```

Comment:
If we know the map_type and the scale, can we not automatically generate the into function, rather than having to separately specify the function?

Reply:
Done